src/platform/lumin-runtime/elements/properties/color-property.js

```diff
@@ -0,0 +1,34 @@
+import { PropertyDescriptor } from './property-descriptor.js';
+import chroma from 'chroma-js';
+
+// Copyright (c) 2019 Magic Leap, Inc. All Rights Reserved
+
+class ColorProperty extends PropertyDescriptor {
+  // Value is:
+  //   string: 'hotpink'; '#ff3399'; 'F39'
+  //   number: 0xff3399
+  //    array: [r, g, b, a?]
+  //   object:
+  //    { h:120, s:1, l:0.75 }
+  //    { l:80, c:25, h:200 }
+  //    { c:1, m:0.5, y:0, k:0.2}
+  //    { r:255, g:165, b:0, a: 0.8 }
+  validate (value) {
+    return ColorProperty.validate(value);
+  }
+
+  static validate (value) {
+    return chroma.valid(value);
+  }
+
+  parse (value) {
+    return ColorProperty.parse(value);
+  }
+
+  static parse (value) {
+    const [r, g, b, a] = chroma(value).rgba(false);
+    return [r / 255, g / 255, b / 255, a];
```

Comment:
[minor] I don't know how js compiler works (how it optimizes the code) but here we divide by 255 three times. It could be more efficient if we multiply by 0.003921568627 instead of dividing - especially that parse is being cold quite often.

Reply:
In terms of optimization you might be right. However in terms of readability I believe `x / 255` is easier to understand.